lib/biran/configurinator.rb

```diff
@@ -103,11 +103,12 @@ def local_config_file_contents
       @local_config_contents = process_config_file(local_config_file)
     end
 
-    def get_secret_contents(app_config)
+    def get_secrets_content(app_config)
```

Comment:
Name change only because I was in here and it was bugging me. File named `secrets` so we get its content.